_parts/part03.md

```diff
@@ -158,6 +162,7 @@ Let's try add the following binary numbers $$101_2$$ and $$1_2$$. The first thin
 I will paste the same exact table as from above but for this example, with the difference being that this is binary arithmetic instead of decimal arithmetic. You can verify by converting the two values to decimal and then asserting that the sum of their decimal values is the same value you get when you convert their binary sum, $$110_2$$, to decimal.
 
 | Computing X+Y  |   |   |   |
+| Carry-Over | | 1 | |
```

Comment:
Same comment above here. Now that I'm rereading my original writing im seeing that what I wrote does not make clear sense (i.e. the carry row shouldn't that be No Yes No) but also even then do you think this row should be boolean values or binary values?